packages/video_player/video_player/lib/video_player.dart

```diff
@@ -545,7 +560,7 @@ class _VideoAppLifeCycleObserver extends Object with WidgetsBindingObserver {
   final VideoPlayerController _controller;
 
   void initialize() {
-    WidgetsBinding.instance.addObserver(this);
+    WidgetsBinding.instance!.addObserver(this);
```

Comment:
Can `WidgetsBinding.instance` be null?

Reply:
no. `runApp()` calls `WidgetsFlutterBinding.ensureInitialized()`.